### PR TITLE
Replace Connect() with Bind()

### PR DESCRIPTION
**Pull request by vadz**
This is something we should have done a long time ago, but better late than never...

### CODE REVIEW

File: src/propgrid/propgrid.cpp [hunk 1]
```diff
@@ -1240,9 +1233,7 @@ void wxPropertyGrid::OnTLPChanging( wxWindow* newTLP )
         if ( newTLP != m_tlpClosed ||
              m_tlpClosedTime+250 < currentTime )
         {
-            newTLP->Connect( wxEVT_CLOSE_WINDOW,
-                             wxCloseEventHandler(wxPropertyGrid::OnTLPClose),
-                             NULL, this );
+            newTLP->Unbind(wxEVT_CLOSE_WINDOW, &wxPropertyGrid::OnTLPClose, this);
```
Comment (MaartenBent) [inline src/propgrid/propgrid.cpp]: Bind

**Review by MaartenBent — COMMENTED** [review]
Noticed one mistake. Also some places where `item->GetId()` was omitted, but I guess this is fine in combination with `item->Bind(...)`.
The rest is caught by CI.

File: samples/docview/doc.cpp [hunk 2]
```diff
@@ -193,13 +193,7 @@ bool wxTextDocument::OnCreate(const wxString& path, long flags)
...
     // subscribe to changes in the text control to update the document state
     // when it's modified
-    GetTextCtrl()->Connect
-    (
-        wxEVT_TEXT,
-        wxCommandEventHandler(wxTextDocument::OnTextChange),
-        NULL,
-        this
-    );
+    GetTextCtrl()->Bond(wxEVT_TEXT, &wxTextDocument::OnTextChange, this);
```
Comment (cecilios) [inline samples/docview/doc.cpp]: Bind ?

**Comment by vadz** [comment]
Thanks to both of you for reviewing this!

**Pull request merged**